[Workers] Add changelog entry for Workers Metrics request distribution map (#19982)
cloudflare/cloudflare-docs: src/content/changelog/workers/2025-02-13-workers-metrics-request-distribution-map.mdx

---
title: Request distribution map added to Workers Metrics
description: View Worker request throughput by location with an interactive map.
products:
  - workers

> date: 2025-02-13 00:00:00 UTC

Reviewer: if we're releasing this now, date should reflect that

---

![Request distribution map](~/assets/images/workers/workers-metrics-request-distribution-map.png)

> The [Workers Metrics dashboard](https://dash.cloudflare.com/?to=/:account/workers/services/view/:worker/production/metrics/) now includes an interactive request distribution map that shows Worker request throughput by location.

Reviewer 2: ```suggestion
The [Workers Metrics dashboard](https://dash.cloudflare.com/?to=/:account/workers/services/view/:worker/production/metrics/) now includes an interactive map that shows which Cloudflare locations served incoming requests to your Worker.
```
